bank-balance-exactly-once/src/main/java/com/github/simplesteph/udemy/kafka/streams/BankBalanceExactlyOnceApp.java

```diff
@@ -33,18 +38,17 @@ public static void main(String[] args) {
 
         // Exactly once processing!!
         config.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, StreamsConfig.EXACTLY_ONCE);
-
+        
         // json Serde
         final Serializer<JsonNode> jsonSerializer = new JsonSerializer();
         final Deserializer<JsonNode> jsonDeserializer = new JsonDeserializer();
         final Serde<JsonNode> jsonSerde = Serdes.serdeFrom(jsonSerializer, jsonDeserializer);
-
-
-        KStreamBuilder builder = new KStreamBuilder();
-
-        KStream<String, JsonNode> bankTransactions =
-                builder.stream(Serdes.String(), jsonSerde, "bank-transactions");
-
+        
+        StreamsBuilder builder = new StreamsBuilder();
+        
+        KStream<String, JsonNode> bankTransactions = builder.stream("bank-transactions");
```

Comment:
you dropped the serializers here, this will fail at runtime

use this instead:
```
KStream<String, JsonNode> bankTransactions = builder.stream("bank-transactions", 
                Consumed.with(Serdes.String(), jsonSerde));
```